### ISSUE TITLE
Xcode 8.3 Image: Can’t Run macOS UI Tests

### ISSUE THREAD

**Issue by SlaunchaMan** [travis-ci/travis-ci#7536]
A [recent build](https://travis-ci.org/SlaunchaMan/Clusterfuck/jobs/216098040) of mine failed with this error when trying to run a macOS UI Test:

```Unable to run UI Tests because Xcode Helper does not have permission to use Accessibility. To enable UI testing, go to the Security & Privacy pane in System Preferences, select the Privacy tab, then select Accessibility, and add Xcode Helper to the list of applications allowed to use Accessibility```

I don’t think this can be scripted, for security reasons, so this would have to be done manually in the image.

**Comment by solarce** [comment]
I believe that I've fixed this in a new 8.3 image, but since it's something I had to try building an open source project to get the _Xcode Helper_ to run and become an option in the Accessibility pane, it could use some testing.

The fix is included in `osx_image: xcode8.3simfix`, which is primarily for https://github.com/travis-ci/travis-ci/issues/7532#issuecomment-290117745.

Could you please test building your project with the new image and let me know how it works?

**Comment by SlaunchaMan** [comment]
[Trying now!](https://travis-ci.org/SlaunchaMan/Clusterfuck/jobs/218479305)

**Comment by solarce** [comment]
@SlaunchaMan based on that job passing, it seems like things are good? 

Also, thanks for your project being open source, it's what I used to trigger _Xcode Helper_ running, it was very helpful to have for testing.

**Comment by SlaunchaMan** [comment]
Yes, this is working now thanks!

**Comment by solarce** [comment]
Awesome, thanks
